### PR TITLE
Add IT Quotes

### PR DESCRIPTION
**Pull request by victorlaerte**
**https://github.com/victorlaerte/awesome-it-quotes**

**This is an list of awesome IT quotes. The aims is to get together all relevant quotes said over the history of IT. There's no much data yet, but I'm still selecting new ones.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by davisonio** [comment]
Please read the requirements above - this can be reopened in 30 days and when all above points are met.

**Comment by victorlaerte** [comment]
@sindresorhus Sorry about other PR, but I created it because to adjust commit message.
I think all points are now met.

**Comment by davisonio** [comment]
It hasn't been around for 30 days yet for a start

**Comment by victorlaerte** [comment]
@davisonio Can it be reopened now?

File: readme.md [hunk 1]
```diff
@@ -328,6 +328,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Fantasy](https://github.com/RichardLitt/awesome-fantasy)
 - [Podcasts](https://github.com/guipdutra/awesome-geek-podcasts)
 - [Email Newsletters](https://github.com/vredniy/awesome-newsletters)
+- [IT Quotes](https://github.com/victorlaerte/awesome-it-quotes)
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by sindresorhus** [comment]
> If you use a license badge, it should be SVG, not PNG

**Comment by victorlaerte** [comment]
Already done.

**Comment by sindresorhus** [comment]
`awesome-it-quotes` => `Awesome IT Quotes`

The badge should be on the right side of the title.

Drop the Awesome image. It's only useful if you actually integrate the list title with the Awesome logo, like https://github.com/vuejs/awesome-vue

The list tagline should be in normal font size and not bold.

Contribution guidelines should be in a separate `CONTRIBUTING.md` file.

File: readme.md [hunk 2]
```diff
@@ -328,6 +328,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Fantasy](https://github.com/RichardLitt/awesome-fantasy)
 - [Podcasts](https://github.com/guipdutra/awesome-geek-podcasts)
 - [Email Newsletters](https://github.com/vredniy/awesome-newsletters)
+- [IT Quotes](https://github.com/victorlaerte/awesome-it-quotes) - Best IT quotes over the history.
```
Comment (sindresorhus) [inline readme.md]: Needs a better description. And `Best` is already implied. This is an awesome list.

**Comment by victorlaerte** [comment]
All topics of the first comment has been done, but the following one I could not understand.
Which one are you saying? 
> The list tagline should be in normal font size and not bold.

## Quotes ?
or table header?
I looked for the bold markdown tag * * and there's no one in md file.

**Comment by sindresorhus** [comment]
> Which one are you saying?

That's because you removed it in https://github.com/victorlaerte/awesome-it-quotes/commit/5355bed795b5e95078fa0e71fdd8e226428663b6#diff-04c6e90faac2675aa89e2176d2eec7d8

You still need a tagline/intro under the list heading. For example: https://github.com/jbmoelker/progressive-enhancement-resources

**Comment by victorlaerte** [comment]
Is it ok now ?

**Pull request merged**